packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -131,7 +134,7 @@ Uploading...
       // Show the current bot in credentials at top of the list
       let first_id = this.botonicApiService.bot.id
```

Comment:
Why is everything in camelCase (or so I thought) and this thing is in snake_case? Again not sure if you should even care about this but yeah...